cxx4opencl/address_spaces.txt

```diff
@@ -343,10 +343,23 @@ their declaration. Developers are free to define a special member function
 that can initialize local address space objects after their declaration. Any
 default values provided for the initialization of members in a class
 declaration are ignored when creating the local address space objects.
-Classes that are instantiated with local address space objects can not
-have references as their members. The same restriction applies recursively
-to the members and base classes. Destructors of local address space objects
-are not invoked automatically. They can be called manually if required.
+Destructors of local address space objects are not invoked automatically.
```

Comment:
FYI this wording change is being reviewed in https://github.com/KhronosGroup/OpenCL-Docs/pull/509